operators/benchmark-receiver/run.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+from interactem.core.logger import get_logger
+from interactem.core.models.messages import BytesMessage
+from interactem.operators.operator import operator
+
+logger = get_logger()
+
+@operator
+def receive_benchmark_frame(
+    inputs: BytesMessage | None, parameters: dict[str, Any]
+) -> None:
+
+    if inputs:
+        meta = inputs.header.meta if hasattr(inputs.header, 'meta') else {}
+        content_preview = inputs.data[:64]  # Show first 64 bytes
+        logger.info(f"Received message: {len(inputs.data)} bytes, meta: {meta}, preview: {content_preview!r}")
```

Comment:
We don't want to log every message, since it can impact how quickly we will be receiving the data. We want to make sure that we are benchmarking how quickly messages of varying sizes are passed between operators.